spec/appcenter_upload_spec.rb

```diff
@@ -598,7 +603,7 @@ def stub_add_to_destination(status, app_name = "app", owner_name = "owner", dest
       stub_create_release_upload(200)
       stub_upload_build(200)
       stub_update_release_upload(200, 'committed')
-      stub_update_release(200)
+      stub_update_release(200, 'No changelog given')
```

Comment:
I think it would have been much easier to change the default value in the function here :)